src/components/project/form-access.vue

```diff
@@ -34,7 +34,7 @@ except according to the terms contained in the LICENSE file.
     <template v-if="dataExists">
       <project-form-access-table :changes-by-form="changesByForm"
         @update:state="updateState"
-        @update:field-key-access="updateFieldKeyAccess"
+        @update:fieldKeyAccess="updateFieldKeyAccess"
```

Comment:
Actually, would you mind renaming this back to `@update:field-key-access` here? Just here in the template and in the template in `ProjectFormAccessTable`, but not anywhere else in the JS (the change works well in the JS). I think making this camel-case here was a mistake on my part: I ended up reverting it after the Vue 3 linter yelled at me.

Reply:
done!